feat: suzyq documentation (#253)
FyraLabs/devdocs: pages/suzyqboard/flash-firmware.mdx

---
title: Flashing Firmware
description: Flash Firmware using the SuzyQ Board
---

When Flashing Firmware, the SuzyQ Board is no different then using a SuzyQable. Check out [MrChromebox's page on flashing firmware](https://docs.mrchromebox.tech/docs/firmware/manual-flashing.html).

Note that this WILL work on Ultramarine and Fedora. Grab the `flashrom` package from Fedora repositories, and `cbfstool` from [Terra](https://terra.fyralabs.com/).

Other known Distros/package managers that package `flashrom` (as of 1-5-2026, we do not guarantee their functionality):

- AlmaLinux 9
- Alpine
- ALT Linux
- AOSC OS
- Arch Linux
- Artix
- CentOS Stream 10
- Chimera Linux
- CRUX
- Debian
- Deepin
- Devuan
- EPEL 6 through 8
- EuroLinux
- Gentoo Linux
- GNU GUIX
- Homebrew
- Kali Linux Rolling
- NixOS/nixpkgs
- openmamba
- OpenMandriva
- OpenSUSE
- Parrot OS
- Raspian
- Slackware
- Ubuntu
- Void Linux

> 

Reviewer: this list is horrifying what the hell

Author: Then make it better

---
title: Flashing Firmware
description: Flash Firmware using the SuzyQ Board
---

When Flashing Firmware, the SuzyQ Board is no different then using a SuzyQable. Check out [MrChromebox's page on flashing firmware](https://docs.mrchromebox.tech/docs/firmware/manual-flashing.html).

Note that this WILL work on Ultramarine and Fedora. Grab the `flashrom` package from Fedora repositories, and `cbfstool` from [Terra](https://terra.fyralabs.com/).

Other known Distros/package managers that package `flashrom` (as of 1-5-2026, we do not guarantee their functionality):

- AlmaLinux 9
- Alpine
- ALT Linux
- AOSC OS
- Arch Linux
- Artix
- CentOS Stream 10
- Chimera Linux
- CRUX
- Debian
- Deepin
- Devuan
- EPEL 6 through 8
- EuroLinux
- Gentoo Linux
- GNU GUIX
- Homebrew
- Kali Linux Rolling
- NixOS/nixpkgs
- openmamba
- OpenMandriva
- OpenSUSE
- Parrot OS
- Raspian
- Slackware
- Ubuntu
- Void Linux

> For an up-to-date list, check out [Repology](https://repology.org/project/flashrom/versions).

Reviewer: i meant do this instead of the lists

Author: Why not both

Author: Ok there this works

---
title: Flashing Firmware
description: Flash Firmware using the SuzyQ Board
---

> When Flashing Firmware, the Fyra SuzyQ Board is no different then using a SuzyQable, so you can just follow [MrChromebox's WP disabling page](https://docs.mrchromebox.tech/docs/firmware/manual-flashing.html).

Reviewer: ```suggestion
When Flashing Firmware, the SuzyQ Board is no different then using a SuzyQable. Check out [MrChromebox's page on flashing firmware](https://docs.mrchromebox.tech/docs/firmware/manual-flashing.html).
```

> Note that this WILL work in Ultramarine Linux/Fedora, and you may `dnf install` `flashrom` from Fedora repositories, and `cbfstool` from [Terra](https://terra.fyralabs.com/).

Reviewer: ```suggestion
Note that this WILL work on Ultramarine and Fedora. Grab the `flashrom` package from Fedora repositories, and `cbfstool` from [Terra](https://terra.fyralabs.com/).
```
